src/utils.ts

```diff
@@ -175,6 +175,11 @@ export function transformCompactObj(compactObj: TTagCompactTranslations) {
             const msgid = msgids[j];
             const translations = compactObj.contexts[ctx][msgid];
             const [newMsgid, newTranslations] = transformCompactTranslate(msgid, translations);
+            if (process.env.NODE_ENV !== 'production' && newContext[newMsgid]) {
+                throw new Error(
+                    `Wrong data! Context "${ctx}" already have key "${newMsgid}", You should change key "${msgid}" or use different context`,
```

Comment:
That error doesn't seems clear to me, maybe we should rephrase that to something like.

```js
const [newMsgid, newTranslations] = transformCompactTranslate(msgid, translations);
const existingMsgid = newContext[newMsgid];
...
throw new Error(`Duplicate msgid (${msgid} and ${existingMsgid} will be interpreted as the same key)  this potentially can lead to translation loss. Consider using context for one of those msgid`s. See the context doc here - https://ttag.js.org/docs/context.html`)
```